.config/Dockerfile

```diff
@@ -11,10 +19,60 @@ ENV GF_AUTH_BASIC_ENABLED "false"
 # Set development mode so plugins can be loaded without the need to sign
 ENV GF_DEFAULT_APP_MODE "development"
 
+
+LABEL maintainer="Grafana Labs <hello@grafana.com>"
+
+ENV GF_PATHS_HOME="/usr/share/grafana"
+WORKDIR $GF_PATHS_HOME
+
 USER root
 
-# Install chromium
 RUN apk add chromium
 
+# Installing supervisor and inotify-tools
+RUN if [ "${development}" = "true" ]; then \
```

Comment:
When I test it locally, `development` is being set to `True` or `False`, notice the **capitalized**. Is it the same for you?

Unless I add `"${development}" = "True"` condition as well, it does not work

Reply:
I set "true" (not capitalized) at the docker-compose.yaml and it worked fine.

Reply:
I have them set with smalls as well, like `true`, `false`. But inside the container, they are being capitalized. I use podman and podman compose, probably that is the reason.

Reply:
I'm using docker-compose and having no issues. But it sounds like your error is valid and should reported upstream

Reply:
It seems to be a bug from `podman-compose`.